### PR TITLE
refactor:  extracted maxBlockCount to a variable

### PR DESCRIPTION
**Pull request by cg-cnu**
## Description

moved `maxBlockCount` to a separate variable and removed the default valued from the function parameter. I already created a first-timer issue to move the value of the variable to config https://github.com/DAVFoundation/xplore/issues/43

## How Has This Been Tested?

npm test and ui are working fine.

### CODE REVIEW

File: src/lib/blockchain.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ function getBalance(account) {
     });
 }
 
-export const getBlocks = (maxBlockCount = 3) => {
+export const getBlocks = (maxBlockCount) => {
```
Comment (TalAter) [inline src/lib/blockchain.js:82]: This breaks existing functionality.

This function is exported.
Your change means that wherever we now import it we no longer have the default value. Only when using it from getTransactions will it work

File: src/lib/blockchain.js [hunk 2]
```diff
@@ -47,7 +47,8 @@ export const getAccounts = () => {
 
 export const getTransactions = async () => {
   const maxTransactionsCount = 10;
-  const blocks = await getBlocks(10);
```
Comment (TalAter) [inline src/lib/blockchain.js:50]: We were previously taking the last 10 blocks... why did you decide to take only 3?
This change breaks existing functionality.

**Comment by cg-cnu** [comment]
Hey @TalAter Sorry for the mistakes! I will try to be careful next time. 

Do you still want me to extract it to a separate variable ?

This time, I won't change the parameter defaults
```
 export const getBlocks = (maxBlockCount = 3) => { 
```

Extract just the variable to `maxBlockCount`
```
const maxBlockCount = 10;
const blocks = await getBlocks(maxBlockCount);
```

Thanks for your help! 🙂

**Comment by TalAter** [comment]
I don't think that's necessary 👍

**Pull request closed without merging**